src/app/content/components/Attribution.tsx

```diff
@@ -125,6 +125,10 @@ class Attribution extends Component<Props> {
 
   public render() {
     const {book} = this.props;
+    // different attribution text for Physics and Statistics
+    const specialCases = ['cce64fde-f448-43b8-ae88-27705cceb0da', '394a1101-fd8f-4875-84fa-55f15b06ba66'];
```

Comment:
Maybe name this `bookIdsWithTEAAttributionText` and then you can remove the comment above?